### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by straithe**
This PR will replace the current issue template with feature request, question, and technical support request templates. People opening issues on this repository will have to choose between the three when opening an issue, there will not be an option to open an issue without a template. 

Documentation on new issue template system from GitHub: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates

### CODE REVIEW

**Review by martinling — CHANGES_REQUESTED** [review]
The bug report, tech support and question templates look OK as far as I can see from the markdown.

Is there a way to preview what they will look like to the user? I tried merging this PR into the `master` branch on my own fork, but when I go to create an issue there I just get an empty text box.

My main concern is that I think there are far too many questions on the feature request form. As it stands I think it will discourage people from making useful feature suggestions.

The questions also overlap a lot, so if we require them all to be answered I think we will just get responses along the lines of:

> > What feature are you requesting?
>
> I want to be able to do X.
> > What problem are you hoping this feature will solve?
>
> I can't do X.
> > What benefit does this feature provide?
>
> It will let people do X.
> > Who will benefit from the new feature?
>
> Anyone who wants to do X.
> > How do current features fail to solve the problem you are experiencing?
>
> The current features don't support X.

I think it would be fine to have a single free text field for feature requests - "What feature would you like to see and why?".

Looking over past issues tagged `enhacement`, we don't get that many, and the ones we have are generally already pretty clear about what is being asked for. A brief description is sufficient. Often this encourages others to comment and flesh out the idea.

Further information like how many other users will benefit, and what the impact will be, are not things that individual users should be expected to know or report accurately. We need to estimate those things for ourselves.

**Comment by straithe** [comment]
I have not found a way to preview what the templates will look like to the user beyond what the examples in the GitHub documentation look like.You can see an example about 2/3 down the page [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository). 

It looks like there might be way to preview templates [?](https://github.com/community/community/discussions/7039). I'll let you know if I figure out a way to preview the templates. 

I have updated the feature request template as you've suggested. Thanks for your insight!

File: .github/ISSUE_TEMPLATE/technical-support.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      placeholder: Please be detailed in what did happen
+    validations:
+      required: true
+  - type: checkboxes
+    id: operating systems
+    attributes:
+      label: What operating systems are you seeing the problem on?
+      multiple: true
+      options: 
+        - Ubuntu
```
Comment (martinling) [inline .github/ISSUE_TEMPLATE/technical-support.md:41]: I suggest to change this to `Linux` rather than listing only a single distro.

**Review by martinling — APPROVED** [review]
I spotted one typo and have suggested a small change but otherwise I think this is good to go.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      value: |
+        Thank you for taking the time to fill out this bug report!
+  - type: dropdown
+    id: issue type
+    attributes: 
+      label: What type of issue is this? 
+      options:
+        - transient - occurring only once
+        - intermittent - occurring irregularly
+        - permanent - occurring repepatedly
```
[the remark this answers is not in the document]
Comment (straithe) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Done!

File: .github/ISSUE_TEMPLATE/technical-support.md (hunk 1 above)
Comment (straithe) [reply .github/ISSUE_TEMPLATE/technical-support.md]: Done!

**Review by martinling — APPROVED** [review]
Looks good to go now, but I'll have another look after merge since we didn't find a way to preview the forms.

**Comment by straithe** [comment]
Ok. Merging now.

**Comment by martinling** [comment]
I just get an empty text box if I go to create a new issue now.

**Comment by straithe** [comment]
Converting .md files to .yml in new branch. Documentation for the feature
was conflicting on this.

**Pull request merged**